### ISSUE TITLE
Include CSP nonce in Animation Disabler's injected script/style tags

### ISSUE THREAD

**Issue by bensheldon** [teamcapybara/capybara#2542]
## Meta
Capybara Version: 3.35.3

## Expected Behavior

When using `Capybara.disable_animation = true`, Capybara should include a CSP nonce (if present in the Rack headers) in the injected `style`/`script` tags.
 
## Actual Behavior

Capybara's injected `style`/`script` tags do not have a nonce and generate a CSP warning in the browser console. This gets noisy if trying to differentiate between actual CSP problems and ones implicitly created by Capybara's tooling. 

This is what the warnings look like:

```log
SEVERE 2022-04-25 07:41:53 -0700: http://127.0.0.1:53163/good_job/jobs 356 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' 'nonce-tH1qhUo4xxLVHs3tEB3DDA=='". Either the 'unsafe-inline' keyword, a hash ('sha256-siq960R56avcceu3sOQRut74UF3gKA8UsEtsP4txVEs='), or a nonce ('nonce-...') is required to enable inline execution.
```

## Steps to reproduce

I'm dumping Browser console logs (via Selenium) with `page.driver.browser.manage.logs.get(:browser)`.

## Suggested implementation:

The Rack headers available to AnimationDisabler do include a nonce value that could be regexed out of the header value, and then included into the injected HTML.

The Rack header value looks like this:

```ruby
@headers #=> { ...   "Content-Security-Policy"=> "default-src 'none'; connect-src 'self'; base-uri 'none'; font-src 'self'; img-src 'self' data:; object-src 'none'; script-src 'self' 'nonce-jAviMuMisoTisVXjgLoWdA=='; style-src 'self' 'nonce-jAviMuMisoTisVXjgLoWdA=='; form-action 'self'; frame-ancestors 'none'" .... }

# e.g.
content_security_nonce = @headers["Content-Security-Policy"].match(/'nonce-([^']*)/)[1]
```

**Comment by twalpole** [comment]
Implemented in 71770a1b81db9ca2180354bc4735bca91633130f

**Comment by bensheldon** [comment]
@twalpole thank you 🎉
